src/libstd/ffi/c_str.rs

```diff
@@ -336,6 +377,35 @@ impl From<NulError> for io::Error {
     }
 }
 
+impl IntoStringError {
+    /// Consumes this error, returning original `CString` which generated the
+    /// error.
+    #[unstable(feature = "convert", reason = "recently added", issue = "0")]
+    pub fn into_inner(self) -> CString {
```

Comment:
Should this instead be called `into_cstring`? There's some precedence here (`NulError::into_vec` and `FromUtf8Error::into_bytes`) but they're not even consistent so I wasn't sure...